src/sql/src/kafka_util.rs

```diff
@@ -134,7 +149,10 @@ pub fn extract_config(
                 // The range of values comes from `statistics.interval.ms` in
                 // https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
                 ValType::Number(0, 86_400_000),
-            ),
+            )
+            .set_default(Some(
+                chrono::Duration::seconds(1).num_milliseconds().to_string(),
+            )),
```

Comment:
(this seems like a rather roundabout way to spell "1000", but leave it if you feel it is clearer as is)